src/Rollbar.js

```diff
@@ -87,49 +90,34 @@ export class Client {
   }
 
   setPerson = (id, name, email) => {
-    this.rollbar.setPerson({id, name, email});
+    this.rollbar.configure({}, {person: {id, name, email}});
```

Comment:
`setPerson` and `clearPerson` in Rollbar.js wraps `configure()` as shown, but only for RN. Calling `configure()` directly allows both to work the same.